arcade/physics_engines.py

```diff
@@ -313,6 +313,10 @@ def __init__(self,
         self.allowed_jumps: int = 1
         self.allow_multi_jump: bool = False
 
+    def update_ladders(self, ladders: Union[SpriteList, Iterable[SpriteList]]):
+        """ Update the current ladder list. """
+        self.ladders = [ladders] if isinstance(ladders, SpriteList) else list(ladders)
+
```

Comment:
Hm, should this instead be done as a property?

Reply:
Ah ok, so it would make it more intuitive for users who go to set the ladder list. 

`    def _get_ladders(self):
        return self._ladders

    def _set_ladders(self, ladders: Union[SpriteList, Iterable[SpriteList]]):
        if ladders:
            self._ladders = [ladders] if isinstance(ladders, SpriteList) else list(ladders)
        else:
            self._ladders = None
    
    def _del_ladders(self):
        self._ladders = None

    ladders = property(
        fget=_get_ladders,
        fset=_set_ladders,
        fdel=_del_ladders,
        doc="The ladder list of the physics engine."
    )`

What are your thoughts on something more like this?

Reply:
Or something more concise with decorators like below:

```
@property
    def ladders(self):
        return self._ladders

@ladders.setter
def ladders(self, ladders: Union[SpriteList, Iterable[SpriteList]]):
    if ladders:
        self._ladders = [ladders] if isinstance(ladders, SpriteList) else list(ladders)
    else:
        self._ladders = None

@ladders.deleter
def ladders(self):
    self._ladders = None
```